src/main/java/Calculator.java

```diff
@@ -0,0 +1,64 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Calculator {
+    static private final String ADD_OPERATION = "+";
+    static private final String SUBTRACT_OPERATION = "-";
+    static private final String MULTIPLY_OPERATION = "*";
+    static private final String DIVIDE_OPERATION = "/";
+    static private final int NUMBER_INDEX = 0;
+    static private final int OPERATION_INDEX = 1;
```

Comment:
```suggestion
    private static final String ADD_OPERATION = "+";
    private static final String SUBTRACT_OPERATION = "-";
    private static final String MULTIPLY_OPERATION = "*";
    private static final String DIVIDE_OPERATION = "/";
    private static final int NUMBER_INDEX = 0;
    private static final int OPERATION_INDEX = 1;
```
`private`는 접근제어자입니다. 모든 선언에서 접근제어자는 맨 처음으로 나와주어야 합니다.